Update to react-dsfr 0.5.3

desoindx opened the pull request.

sandrica89 commented:
Focus on tab panel are not positionned properly. This is probably caused by the `padding: 0` I added on it to avoid loosing too much space because there are already many space with successive margins. What do you think ? Should we leave it like that or remove the padding?

![Selection_009](https://user-images.githubusercontent.com/68006779/127451749-7216d9c5-8d50-4e8d-a630-0e724345a06a.jpg)

desoindx commented:
> Focus on tab panel are not positionned properly. This is probably caused by the `padding: 0` I added on it to avoid loosing too much space because there are already many space with successive margins. What do you think ? Should we leave it like that or remove the padding?
> 
> ![Selection_009](https://user-images.githubusercontent.com/68006779/127451749-7216d9c5-8d50-4e8d-a630-0e724345a06a.jpg)

Fixed

sandrica89 commented on frontend/src/components/PsyListing/PsyTable.jsx:
```diff
@@ -69,54 +108,10 @@ const PsyTable = ({
-            </thead>
-            <tbody>
-              {psychologists
-                .slice((page - 1) * 10, page * 10)
```
No need to "slice" anymore? With `data` prop, does `Table` handle pagination internaly? If yes, how is it linked with the `Pagination` component?

desoindx replied:
good catch !

sandrica89 commented on frontend/src/App.css:
```diff
@@ -129,5 +129,6 @@
 @media (min-width: 48em) {
   .faqPage .fr-tabs__panel {
     padding: 0; 
+    outline-offset: 0;
```
:clap:
